tools/releases/docker.sh

```diff
@@ -0,0 +1,150 @@
+#!/usr/bin/env bash
+
+set -e
+
+KUMA_DOCKER_REPO="kong-docker-kuma-tests-docker.bintray.io"
```

Comment:
Do we need to change this before merging?

Reply:
We do